index.js

```diff
@@ -11,7 +8,7 @@ const file = process.argv[2];
 
 fs.mkdir(pathName, { recursive: true }, (error) => {
     if(error) {
-        console.log('failed to make dir')
+        throw error;
```

Comment:
Only log an error without throwing if the issue won't cause your code to behave badly. In this case we NEED the dir, so best to throw the error and bail out rather than continue and bump into a bug later that was originally instigated here.